java.lang.RuntimeException at openfoodfacts.github.scrachx.openfood.utils.WifiUploadReceiver.onReceive

```java.lang.RuntimeException: 
  at android.app.ActivityThread.handleReceiver (ActivityThread.java:3695)
  at android.app.ActivityThread.-wrap18 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1969)
  at android.os.Handler.dispatchMessage (Handler.java:108)
  at android.os.Looper.loop (Looper.java:166)
  at android.app.ActivityThread.main (ActivityThread.java:7415)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:245)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:921)
``` 
```
Caused by: java.lang.IllegalStateException: 
  at android.app.ContextImpl.startServiceCommon (ContextImpl.java:1701)
  at android.app.ContextImpl.startService (ContextImpl.java:1657)
  at android.content.ContextWrapper.startService (ContextWrapper.java:644)
  at android.content.ContextWrapper.startService (ContextWrapper.java:644)
  at openfoodfacts.github.scrachx.openfood.utils.WifiUploadReceiver.onReceive (WifiUploadReceiver.java:29)
  at android.app.ActivityThread.handleReceiver (ActivityThread.java:3685)
  at android.app.ActivityThread.-wrap18 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1969)
  at android.os.Handler.dispatchMessage (Handler.java:108)
  at android.os.Looper.loop (Looper.java:166)
  at android.app.ActivityThread.main (ActivityThread.java:7415)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:245)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:921)
```

--- a/app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/WifiUploadReceiver.java
+++ b/app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/WifiUploadReceiver.java
@@ -1,5 +1,6 @@
 package openfoodfacts.github.scrachx.openfood.utils;
 
+import android.app.Notification;
 import android.app.NotificationManager;
 import android.app.PendingIntent;
 import android.app.Service;
@@ -11,8 +12,6 @@ import android.os.Handler;
 import android.os.IBinder;
 import android.support.annotation.Nullable;
 import android.support.v4.app.NotificationCompat;
-import android.app.Notification;
-import android.os.Build;
 
 import java.util.List;
 
@@ -32,7 +31,7 @@ public class WifiUploadReceiver extends BroadcastReceiver {
     public void onReceive(Context context, Intent intent) {
 
         int wifiState = intent.getIntExtra(WifiManager.EXTRA_WIFI_STATE, -1);
-        if (WifiManager.WIFI_STATE_CHANGED_ACTION.equals(intent.getAction()) && WifiManager.WIFI_STATE_ENABLED == wifiState) {
+     /*   if (WifiManager.WIFI_STATE_CHANGED_ACTION.equals(intent.getAction()) && WifiManager.WIFI_STATE_ENABLED == wifiState) {
 
             //context.startService(new Intent(context, WifiService.class));
             // Quick fix. TODO: Fix it the right way: https://github.com/openfoodfacts/openfoodfacts-androidapp/issues/1583
@@ -42,7 +41,8 @@ public class WifiUploadReceiver extends BroadcastReceiver {
                 context.startService(new Intent(context, WifiService.class));
             }
             
-        }
+        }*/
+        //Above code commented out by ross-holloway94 18 June 2018. Refer to issue #1583
 
     }
 

